openwisp_ipam/tests/test_api.py

```diff
@@ -251,6 +251,24 @@ def test_hosts_list_api(self):
         self.assertEqual(self.client.get(response.data['previous']).status_code, 200)
         self.assertIsNone(response.data['next'])
 
+    def test_bearer_auth(self):
+        self._create_user(username='tester', password='tester')
```

Comment:
make this user is a superuser so that he can have all the permission needed to perform the required operation. that is this should be `self._create_user(username='tester', password='tester', is_superuser=True)`